frontend/src/stories/ActionBarNoSIdeOption.example.html

```diff
@@ -0,0 +1,16 @@
+<div class="spot-action-bar">
+  <div class="spot-action-bar--right">
+    <button
+      type="button"
+      class="spot-action-bar--action button"
+    >
+      <span [textContent]="'Cancel'"></span>
```

Comment:
Any reason not to just write `<button ...>Cancel</button>`?

Reply:
Nope, leftover of removing a previously existing icon. Does it cause any kind of confusion?

Reply:
Yeah I prefer for examples to only include the bare minimum, since they'll be copy pasted a lot :P

Reply:
kk, will strip them down